impl_core/src/test/java/io/opencensus/implcore/trace/export/NoopRunningSpanStoreImplTest.java

```diff
@@ -66,17 +66,16 @@
   @Before
   public void setUp() {
     MockitoAnnotations.initMocks(this);
-    SpanImpl span =
-        SpanImpl.startSpan(
-            spanContext,
-            recordSpanOptions,
-            SPAN_NAME,
-            null,
-            false,
-            TraceParams.DEFAULT,
-            startEndHandler,
-            timestampConverter,
-            testClock);
+    SpanImpl.startSpan(
```

Comment:
I'm not sure I understand this change.  Does creating the span have a side effect?  I think it is more likely that the new span should be assigned to `spanImpl`, which is null.

I also wonder if RunningSpanStoreImpl.onStart and onEnd should have rejected the null `spanImpl`.

Reply:
I don't have context on this test. The local variable `SpanImpl span` is never used and failed the build when import. 

@HailongWen should this be assigned to `private SpanImpl spanImpl;` instead?

Reply:
I think so.

Reply:
Done.